src/Sylius/Bundle/CoreBundle/Migrations/Version20240405121345.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240405121345 extends AbstractMigration
```

Comment:
Seeing that the syntax for creating an index on prostgres is the same. I just extended the postgres migration.

Reply:
@Rafikooo I've changed the base of the PR to 2.0 and use an `AbstractPostgreSQLMigration`.

Reply:
Ah, sorry I missed this thread, but You'll either need to also create one for Mysql as well or revert to `Doctrine\Migrations\AbstractMigration` since the syntax is the same.

Only extending `AbstractPostgreSQLMigration` will result in this migration skipping on Mysql and its counterpart from Sylius will skip Postgres instead.